hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2167,6 +2167,7 @@ private class Emit(
         val jet = new Emit(ctx, joinF, 1).emit(Subst(join, BindingEnv(Env[IR](r -> In(0, rEltPType)))), fenv, er, container)
         joinF.emit(Code(jet.setup, jet.m.mux(Code._fatal("ArrayLeftJoinDistinct: joined can't be missing"), jet.v)))
 
+        //FIXME: Remove region?
```

Comment:
Still don't entirely follow this code, so wanted to check.

Reply:
yes, the changes here are correct. remove comment?